DOCS - Repeating panels require variable to have one or more items (#23080)
grafana/grafana: docs/sources/reference/templating.md

Only make changes to the first panel (the original template). To have the changes take effect on all panels you need to trigger a dynamic dashboard re-build.
You can do this by either changing the variable value (that is the basis for the repeat) or reload the dashboard.

> Repeating panels require variables to have one or more items selected; you can not repeat a panel zero times to hide it.

Reviewer: ```suggestion
> **Note:** Repeating panels require variables to have one or more items selected; you cannot repeat a panel zero times to hide it.
```
